README.md

```diff
@@ -42,6 +43,11 @@ A crowd sourced list of the best music to program to! Please contribute and feel
 |Bag Raiders|Shooting Stars|Song|[link](https://open.spotify.com/track/0UeYCHOETPfai02uskjJ3x?si=ThRZUizATyy2S0ZMjezQ4g)|
 |Various|WIP (Work in Progress)|Playlist|[link](https://open.spotify.com/user/bwandin/playlist/2xWIxCqVb0gPK6EQMbtmqr?si=p4qwlX-_S6eqvTJZQQyECQ)|
 
+## Folk
+|Artist|Name|Type|Link|
+|---|---|---|---|
+|Jay Ungar|Ashokan Farewell|Song|[link](https://open.spotify.com/track/43JiVtB9jSv89httTl3Hjb?si=g1JGIwMtT8mRGQ_j5OPKXg)
```

Comment:
Please use correct Markdown syntax. You are missing the | at the end of your row in the table.